pkgs/applications/networking/instant-messengers/gtkcord/default.nix

```diff
@@ -0,0 +1,45 @@
+{ makeDesktopItem, lib, buildGoModule, fetchFromGitHub, pkgconfig, go, gnome3, libhandy }:
+
+buildGoModule rec {
+  pname = "gtkcord";
+  version = "0.0.4";
+
+  src = fetchFromGitHub {
+    owner = "diamondburned";
+    repo = "gtkcord3";
+    rev = "v${version}";
+    sha256 = "0n9600s50zpx8kv89icrlxarm3633gnizg1sq1zpask8bbv2xnfd";
+  };
+
+  nativeBuildInputs = [ pkgconfig go ];
+  
+  buildInputs = [ gnome3.glib gnome3.gtk libhandy ];
+
+  desktopFile = makeDesktopItem {
+		name = pname;
+    desktopName = "gtkcord";
+		exec = "gtkcord3";
+		icon = "gtkcord3";
+		categories = "GTK;GNOME;Chat;";
+	};
+
+	preFixup = ''
+		mkdir -p $out/share/icons/hicolor/256x256/apps/ $out/share/applications/
+		# Install the desktop file
+		cp "${desktopFile}"/share/applications/* $out/share/applications/
+		# Install the icon
+		cp "$src/logo.png" $out/share/icons/hicolor/256x256/apps/gtkcord3.png
+	'';
+
+  vendorSha256 = "0kzq8x0qcjr443ydj05s2f8ybszc2jzqa2annadp3hry8x2va2in";
+
+  subPackages = [ "." ];
+
+  meta = with lib; {
+    description = "A lightweight Discord client which uses GTK3 for the user interface.";
```

Comment:
I believe that usually we don't include the trailing period `.` in descriptions.

Other than that, this pull request looks good to me! I don't have a Discord account, but I confirm that gtkcord builds and starts. Thank you for packaging gtkcord. :-)

Reply:
Correct. Initial “A” can also be omitted.